app/routes/ap-ar-report-listing.js

```diff
@@ -74,11 +74,17 @@ function generateRoutes (reportName, reportDataUrl, reportDataKey) {
                 'Original Invoice Value': data.value,
                 'Invoice Number': data.invoiceNumber,
                 'Invoice Delta Amount': data.deltaAmount,
-                'D365 Invoice Imported': data.routedToRequestEditor,
-                'D365 Invoice Payment': data.settledValue,
-                'PH Error Status': data.phError,
-                'D365 Error Status': data.daxError
+                'D365 Invoice Imported': data.routedToRequestEditor
```

Comment:
A bit pedantic but it's probably neater to have everything in as default in mappedDate, then just say

```
if (reportName === 'ar-listing') {
  delete mappedData['D365 Invoice Payment']
}
```

Reply:
Also no semi colons! :)